### PR TITLE
EPMDJ-10176: fix read notifications

### PR DESCRIPTION
**Pull request by dmitrii-kliushentsev**

### CODE REVIEW

File: src/ui/components/panels/notifications-panel/notifications-panel.tsx [hunk 1]
```diff
@@ -16,18 +16,19 @@
...
 import { deleteAllNotifications, readAllNotifications } from "./api";
 import { Notification } from "./notification";
 import { PanelStub } from "../../panel-stub";
 
-export const NotificationsPanel = ({ isOpen, onClosePanel }: PanelProps) => {
+interface NotificationsPanelProps extends PanelProps {
+  payload: NotificationType[],
+}
+
+export const NotificationsPanel = ({ isOpen, onClosePanel, payload: notifications }: NotificationsPanelProps) => {
```
Comment (IliaSolovev) [inline src/ui/components/panels/notifications-panel/notifications-panel.tsx:30]: If we will receive a new notification it won`t display here

**Review by IliaSolovev — CHANGES_REQUESTED** [review]

File: src/ui/pages/agent/agent-page.tsx [hunk 2]
```diff
@@ -35,16 +35,18 @@ export const AgentPage = () => {
   const { systemSettings } = useActiveBuild(agent?.id) || {};
   const [activeBuildInfo] = useAdminConnection<AgentBuildInfo[]>(`/api/agent/${agent.id}/builds`) || [];
   const setPanel = useSetPanelContext();
-  const notifications =
-    useAdminConnection<Notification[]>("/notifications") || [];
-  const newBuildNotification =
-    notifications.find((notification) => notification.agentId === agentId) ||
-    {};
+  const notifications = useAdminConnection<Notification[]>("/notifications") || [];
```
Comment (IliaSolovev) [inline src/ui/pages/agent/agent-page.tsx:38]: move it to t2c because build it is a t2c entity

File: src/ui/pages/agent/agent-page.tsx [hunk 3]
```diff
@@ -86,3 +88,7 @@ async function readNotification(
     onError && onError(message as string);
   }
 }
+
+function filterNotificationForVersion(notifications: Notification[], version?: string) {
```
Comment (RomanAchilov) [inline src/ui/pages/agent/agent-page.tsx:92]: the fact that this is a filter is clear by the method from the array

Comment (RomanAchilov) [inline src/ui/pages/agent/agent-page.tsx:92]: mb
```suggestion
function getNotificationsByBuildVersion(notifications: Notification[], version?: string) {
```

**Review by RomanAchilov — CHANGES_REQUESTED** [review]

**Review by RomanAchilov — APPROVED** [review]

**Review by IliaSolovev — APPROVED** [review]

**Pull request merged**